public/js/src/frame.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+class Frame {
+  constructor(){
+    this.currentFrame = [];
+    this.finalFrame = false;
+  };
+
+  currentFrame(){
+    return this.currentFrame;
+  };
+
+  addToFrame(pins){
```

Comment:
Is 'addToFrame' a bit redundant here as the class is already called Frame? I feel the same about the name of your 'currentFrame' array. The name Frame relates to both the class but also the array within the class. Not a big deal but something to think about